recipes/pynvjitlink/meta.yaml

```
@@ -0,0 +1,71 @@
+{% set name = "pynvjitlink" %}
+{% set version = "0.1.14" %}
+
+package:
+  name: pynvjitlink
+  version: {{ version }}
+
+source:
+  url:  https://github.com/rapidsai/pynvjitlink/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b1991a211df1c33b2a6e99ea8eb5cea34c0efaf5317fe04f1fe26183d8a5552e
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [cuda_compiler_version == "None"]
+  skip: true  # [(cuda_compiler_version or "").startswith("11")]
+  skip: true  # [py<39]
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - {{ stdlib('c') }}
+    - cmake >=3.26.4
+    - make
+  host:
+    - cuda-version {{ cuda_compiler_version }}
+    - libnvjitlink-dev
+    - libnvjitlink-static
+    - python
+    - pip
+    - scikit-build-core
+  run:
+    - python
+    - numba >=0.57
+    - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
```

Probably need to adjust this pinning so that the minimum version is `>=x.x.x`

```suggestion
    - {{ pin_compatible('cuda-version') }}
```